oca_dependencies.txt

```diff
@@ -1,2 +1,3 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
 web
+rma
```

Comment:
@solo4games you can remove this due to #3296

Reply:
Done, check pls